### PR TITLE
Use interactive analysis

### PR DESCRIPTION
**Pull request by jason-seqera**
Would need to apply to 24.2 as well if approved.

Applying on top of 24.2 docs port, to be rebased to master.

### CODE REVIEW

File: platform_versioned_docs/version-24.1/enterprise/kubernetes.mdx [hunk 1]
```diff
@@ -252,7 +252,7 @@ The initContainers will wait until both the Seqera and pipeline optimization ser
 
 ### Data Studios
 
-[Data Studios](../data_studios/index.mdx) is an interactive tertiary analysis environment available in organizational workspaces. To enable Data Studios, see [Data Studios deployment](./data-studios.mdx).
+[Data Studios](../data_studios/index.mdx) is an interactive interactive analysis environment available in organizational workspaces. To enable Data Studios, see [Data Studios deployment](./data-studios.mdx).
```
Comment (pinin4fjords) [inline platform_versioned_docs/version-24.1/enterprise/kubernetes.mdx:255]:
```suggestion
[Data Studios](../data_studios/index.mdx) is an interactive analysis environment available in organizational workspaces. To enable Data Studios, see [Data Studios deployment](./data-studios.mdx).
```

**Review by justinegeffen — COMMENTED** [review]
This looks good to me - my only comment/question is whether we can scale back on the number of times we use "interactive". If there are instances where "analysis" would make sense on its own or can be replaced with something less repetitive, we should do that. 

The use of "tertiary" (and now "interactive") does lean more toward marketing/sales terminology and I don't think we need to lean as heavily into that while retaining meaning. @robnewman, is this fair?

**Review by robnewman — APPROVED** [review]
Looks good!

**Comment by robnewman** [comment]
> This looks good to me - my only comment/question is whether we can scale back on the number of times we use "interactive". If there are instances where "analysis" would make sense on its own or can be replaced with something less repetitive, we should do that.
> 
> The use of "tertiary" (and now "interactive") does lean more toward marketing/sales terminology and I don't think we need to lean as heavily into that while retaining meaning. @robnewman, is this fair?

I wouldn't sweat this too much right now. Let's just make the change. We can review in more detail when we go to GA

**Pull request merged**